option/internaloption/internaloption.go

```diff
@@ -209,6 +209,27 @@ func (w enableNewAuthLibrary) Apply(o *internal.DialSettings) {
 	o.EnableNewAuthLibrary = bool(w)
 }
 
+// EnableAsyncRefreshDryRun  returns a ClientOption that specifies if libraries in this
+// module should asynchronously refresh auth token in parallel to sync refresh
```

Comment:
Can you add more information here explaining why it is a "dry run" and what the goal or purpose is?

Reply:
Done